### PR TITLE
Niten import

### PR DESCRIPTION
**Pull request by evemontalvao**
**DESCRIPTION**

We were having the following problem: in Gaiden, Niten import was done like `@import 'node_modules/@getninjas/niten-tokens/build/web/niten-tokens';` in `gaiden.scss` file and the files where the variables are used. This is bad because we needed an `import` in each file and also because in tanya, when we imported the `_colors.scss` css for example, the path `../../../node_modules/@getninjas/niten...` doesn't exist anymore, since Gaiden is also a module it doesn't have `node_modules`. So I created a script that can be run on the terminal to import Niten css into any project. Niten must be installed globally and to import into Tanya, for example, the command would be `niten --path" src/static/scss/gaiden"`

**CHANGELOG**

* Niten is now a global package

### CODE REVIEW

File: build/web/_niten-tokens.scss [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
  * Do not edit directly
- * Generated on Tue, 06 Aug 2019 18:40:46 GMT
+ * Generated on Fri, 16 Aug 2019 19:57:44 GMT
  */
 
 $color-brand-first-extra-dark: #000000;
 $color-brand-first-dark: #212121;
 $color-brand-first-medium: #575757;
-$color-brand-first-light: #FFFFFF;
+$color-brand-first-light: #ffffff;
```
Comment (williamcosta) [inline build/web/_niten-tokens.scss]: Needed to put hex colors in lower case?

File: package.json [hunk 2]
```diff
@@ -25,8 +25,12 @@
   "devDependencies": {
     "style-dictionary": "^2.8.1"
   },
+  "bin": {
+    "niten": "./bin/index.js"
```
Comment (williamcosta) [inline package.json]: It is not yet clear how the flow will work.

File: build/web/_niten-tokens.scss (hunk 1 above)
Comment (evemontalvao) [reply build/web/_niten-tokens.scss]: Yes, lint 😄

File: package.json (hunk 2 above)
Comment (evemontalvao) [reply package.json]: On Tanya, for example:
1. `npm i -g @getninjas/niten-tokens`
2. `niten --path "src/static/scss/gaiden"`
3. Import variables using "@import '../../gaiden/niten-tokens"

@williamcosta

Comment (pyymenta) [reply package.json]: > On Tanya, for example:
> 
> 1. `npm i -g @getninjas/niten-tokens`
> 2. `niten --path "src/static/scss/gaiden"`
> 3. Import variables using "@import '../../gaiden/niten-tokens"
> 
> @williamcosta

Should this be in README?

Comment (evemontalvao) [reply package.json]: Yes! I'm gonna open another PR with that. Thanks for remind me 😄

Comment (williamcosta) [reply package.json]: So we don't need these changes?

Comment (evemontalvao) [reply package.json]: We need it so we can install niten globally @williamcosta

**Pull request merged**